paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PrepareResourcesTask.kt

```diff
@@ -32,22 +29,21 @@ import org.gradle.api.tasks.TaskAction
 
 @CacheableTask
 abstract class PrepareResourcesTask : DefaultTask() {
+
   @get:Input
   abstract val packageName: Property<String>
 
-  @get:InputDirectory
-  @get:PathSensitive(PathSensitivity.RELATIVE)
-  abstract val mergeResourcesOutput: DirectoryProperty
+  @get:Input
+  abstract val mergeResourcesOutput: Property<String>
 
   @get:Input
   abstract val targetSdkVersion: Property<String>
 
   @get:Input
   abstract val compileSdkVersion: Property<String>
 
-  @get:InputDirectory
-  @get:PathSensitive(PathSensitivity.RELATIVE)
-  abstract val mergeAssetsOutput: DirectoryProperty
+  @get:Input
+  abstract val mergeAssetsOutput: Property<String>
```

Comment:
```suggestion
  abstract val mergeAssetsOutputDir: Property<String>
```

...since it's now a String property